### ISSUE TITLE
Incompatible argument type: Required type: array. Actual type: \Magento\Framework\Stdlib\BooleanUtils

### ISSUE THREAD

**Issue by koenner01** [shipperhq/module-shipper#32]
We are having compilation problems on production mode:

```
Errors during compilation: ShipperHQ\Shipper\Ui\Component\Listing\Column\DeliveryDate
Incompatible argument type: Required type: array. Actual type: \Magento\Framework\Stdlib\BooleanUtils;
```

We ran into this problem when after a composer update;
The shipper module was updated from 20.6.1 to 20.7.0

**Comment by wsagen** [comment]
Can you let me know the version of Magento you are running

**Comment by piatrouski** [comment]
Same issue - 2.1.7 EE

**Comment by koenner01** [comment]
We are running 2.1.7 CE

**Comment by wsajason** [comment]
A new version has been released to address this, 2.7.1.

**Comment by jmellon** [comment]
If it helps here are my observations: 2.1.8 CE.  Installed manually app/code/~  No compilation errors using this version of Shipper HQ

**Comment by wsagen** [comment]
Closing as this issue is resolved in version 20.7.1
